publish.gradle

```diff
@@ -215,19 +198,6 @@ task cppHeadersZip(type: Zip, dependsOn: make) {
         archiveBaseName = zipBaseName
         duplicatesStrategy = 'exclude'
 
-        manifest {
-            attributes(
-                    "Created-By": "WPILib Gradle Build Script",
-                    "Implementation-Title": "OpenCV Native Libraries, ${project.ext.platformClassifier}",
-                    "Implementation-Version": pubVersion,
-                    "Implementation-Vendor": "Itseez",
-
-                    "Bundle-Name": "${archiveBaseName.get()}",
-                    "Bundle-Version": pubVersion,
-                    "Bundle-License": "https://opensource.org/licenses/BSD-3-Clause",
-                    "Bundle-Vendor": "WPILib")
-        }
-
```

Comment:
What’s the reason to remove these?

Reply:
The manifest blocks are in the middle of the header and source zip tasks, which means they do nothing. There's no manifest in either of the resulting archives.